server/worker.js

```diff
@@ -0,0 +1,22 @@
+const cron = require('node-cron');
+const model = require('./database/queries');
+const Moment = require('moment');
+
+let time;
+let prevJobTime = new Moment().subtract(5, 'years').format('YYYY-MM-DD HH:mm:ss');
```

Comment:
would this function be more efficient if the prevJobtime is shorter, like one day or a week?